packages/salesforcedx-vscode-core/package.json

```diff
@@ -811,6 +823,10 @@
           "dark": "resources/dark/code.svg"
         }
       },
+      {
+        "command": "sfdx.create.manifest",
+        "title": "%force_create_manifest%"
```

Comment:
cool...where does the string replace happen here?  Is it custom or is that something baked into extensions?

Reply:
This is baked into the extensions - it's able to find strings automatically from `package.nls.json`, as shown in this example repo (which I also just confirmed lol): https://github.com/microsoft/vscode-extension-samples/tree/main/i18n-sample

Reply:
🆒